src/daemon/daemon.h

```diff
@@ -24,6 +24,7 @@
 #include <multipass/delayed_shutdown_timer.h>
 #include <multipass/memory_size.h>
 #include <multipass/metrics_provider.h>
+#include <multipass/network_interface.h>
```

Comment:
Makes sense but... it doesn't work. It happens that `NetworkInterface` is used in a `vector`. And it seems the vector needs the size of element. I didn't dig more into this because I have more urgent changes to do. I'll return to this later on.

Reply:
No, you're right. My bad, the vector needs the size of the object.